remo/profiles/models.py

```diff
@@ -178,6 +179,8 @@ class UserProfile(caching.base.CachingMixin, models.Model):
     unavailability_task_id = models.CharField(max_length=256, blank=True,
                                               null=True, editable=False,
                                               default='')
+    is_rotm_nominee = models.BooleanField(choices=ROTM_NOMINEE_CHOICES,
```

Comment:
Since you are defining fields ``choices`` in ``RotmNomineeForm`` this could be omitted.